src/components/visualizations/orbitalViewer/OrbitalDetails.jsx

```diff
@@ -13,7 +13,8 @@ import {
 import Button from '../../site/button/index.js';
 
 function OrbitalDetails({ data, velocity, type }) {
-  const { H, a, i, e, Principal_desig: name, Earth_moid: moid } = data || {};
+  const { H, a, i, e, name, Principal_design: pdName, Earth_moid: moid } =
```

Comment:
Typo: `Principal_design` should be `Principal_desig`.  As is is breaking the name in details pane toward the end of the investigation: pages `classifying-objects`, `classifying-objects-1`, and `classifying-objects-2`.  Alternatively if all the new data uses `name` instead of `Principal_desig` could update the data for the aforementioned pages to use `name` as well.